addons/wowl/static/src/components/navbar/navbar.ts

```diff
@@ -1,19 +1,33 @@
-import { Component, hooks, utils } from "@odoo/owl";
+import { Component, hooks, tags, utils } from "@odoo/owl";
 import { OwlEvent } from "@odoo/owl/dist/types/core/owl_event";
 import { useService } from "../../core/hooks";
 import { MenuTree } from "../../services/menus";
 import { OdooEnv } from "../../types";
 import { Dropdown } from "../dropdown/dropdown";
 import { DropdownItem } from "../dropdown/dropdown_item";
 const { useExternalListener } = hooks;
+const { xml } = tags;
 
 export interface NavBarState {
   selectedApp: null | MenuTree;
 }
 
+class AppMenuDropdownItem extends DropdownItem {
+  static template = xml`
+    <li
+      class="o_dropdown_item"
+      t-att-data-menu-xmlid="props.payload.xmlid"
+      t-on-click="onClick"
+      owl="1"
+    >
+      <span><t t-slot="default" /></span>
+    </li>
```

Comment:
Duplication of the DropdownItem template, at the exception of line 19...
IMHO, this is ugly.

Reply:
use xpath?